torch_tvm/operators.cpp

```diff
@@ -60,21 +78,31 @@ RegisterTVMOperator reg({
        auto out = tvm::relay::CallNode::make(op, add_inputs, tvm::Attrs(), {});
        return out;
      }},
-    {Symbol::fromQualString("aten::conv2d"),
+    {Symbol::fromQualString("aten::_convolution"),
      [](Node* node, tvm::Array<tvm::relay::Expr> inputs) {
-       static const tvm::relay::Op& op = tvm::relay::Op::Get("nn.conv2d");
+       bool is_transpose = relayToConstant<bool>(inputs[6]);
+       // check the operator to emit base on is_transpose
+       auto op = tvm::relay::Op::Get("nn.conv2d");
+       if (is_transpose) {
+          op = tvm::relay::Op::Get("nn.conv2d_transpose");
+       }
+
+       // input and filter
        tvm::Array<tvm::relay::Expr> new_inputs = {
            inputs[0],
            inputs[1],
        };
+       // TODO: support passing bias to tvm
+
        auto conv_attrs = tvm::make_node<tvm::relay::Conv2DAttrs>();
-       conv_attrs->groups = 1;
+       conv_attrs->groups = relayToConstant<int>(inputs[8]);
        conv_attrs->data_layout = "NCHW";
        conv_attrs->kernel_layout = "OIHW";
-       conv_attrs->kernel_size = {3, 3};
-       conv_attrs->padding = {0, 0};
-       conv_attrs->strides = {1, 1};
-       conv_attrs->dilation = {1, 1};
+       // TODO: figure out kernel size usage in tvm
```

Comment:
I feel like the kernel_size arg is redundant (it is optional already) in the [relay api](https://docs.tvm.ai/langref/relay_op.html?highlight=conv2d#tvm.relay.nn.conv2d), because kernel size is mainly used to create the weight parameter but the relay op already passed the weight tensor in, that's also the pytorch semantic as we could see from [conv module implementation](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/conv.py#L37), and the pytorch `aten::_convolution` op does not have kernel size as argument as well.